Project.toml

```
@@ -28,7 +28,7 @@ cohomCalg_jll = "5558cf25-a90e-53b0-b813-cadaa3ae7ade"
 AbstractAlgebra = "0.42.3"
 AlgebraicSolving = "0.5.1"
 Distributed = "1.6"
-GAP = "0.10.2"
+GAP = "0.11.2"
```

```suggestion
GAP = "0.11.3"
```
adapt once https://github.com/oscar-system/GAP.jl/pull/1037 is released